Catappult/appcoins-wallet-android
# [feature/APPC-3399] Simplify onboarding screen

**What does this PR do?**

   Simplifies the onboarding flow, removing the first screen and removing the bottom sheet for the terms and conditions

**Database changed?**

   No

**Where should the reviewer start?**

- OnboardingFragment.kt

**How should this be manually tested?**

  - Open the wallet
  - Test the launch wallet option (should simply create the wallet if no wallet was created)
  - Test the option to recover the wallet

**What are the relevant tickets?**

  https://aptoide.atlassian.net/browse/APPC-3399

**Code Review Checklist**
